frontend/src/shared/components/tasque-dropdown/dropdown.component.sass

```diff
@@ -0,0 +1,85 @@
+@import 'src/colors.sass'
+    
+.dropdown_wrpr
```

Comment:
style naming convention uses '-' instead of '_'
also, you can use full word, it's not that long